gensim/console_api/link.py

```diff
@@ -0,0 +1,118 @@
+# coding: utf8
+
+"""Copyright (C) 2016 ExplosionAI UG (haftungsbeschränkt), 2016 spaCy GmbH,
+2015 Matthew Honnibal
+https://github.com/explosion/spaCy/blob/master/LICENSE
+"""
+
+from __future__ import unicode_literals
+
+import ujson
+import pip
+from pathlib import Path
+import importlib
+from ..compat import unicode_, symlink_to
+from .. import utils
+
+_data_path = Path(__file__).parent.parent / 'data'
+
+
+def get_data_path(require_exists=True):
+    if not require_exists:
+        return _data_path
+    else:
+        return _data_path if _data_path.exists() else None
+
+
+def parse_package_meta(package_path, package, require=True):
+    """
+    Check if a meta.json exists in a package and return its contents as a
+    dictionary. If require is set to True, raise an error if no meta.json
+    found.
+    """
+    location = package_path / package / 'meta.json'
```

Comment:
/ is similar to os.path.join()
https://docs.python.org/3/library/pathlib.html#operators